src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,101 @@
+import { useParams } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import { Person } from '../../types/Person';
+import cn from 'classnames';
+
+enum Properties {
+  Name = 'Name',
+  Sex = 'Sex',
+  Born = 'Born',
+  Died = 'Died',
+  Mother = 'Mother',
+  Father = 'Father',
+}
+
+interface Props {
+  tablePeople: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { tablePeople } = props;
+
+  const { slug: prevSlug } = useParams();
+
+  const findMotherAndFather = (nameOfParent: string | null): Person | null => {
+    return tablePeople.find(person => person.name === nameOfParent) || null;
+  };
+
+  const selectedPeople = tablePeople.map(person => {
+    return {
+      ...person,
+      father: findMotherAndFather(person.fatherName),
+      mother: findMotherAndFather(person.motherName),
+    };
+  });
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {Object.values(Properties).map((proper, index) => (
+            <th key={index}>{proper}</th>
```

Comment:
Using the index as a key in React can lead to issues if the list changes dynamically. It's better to use a unique identifier if available. In this case, since the properties are static, it might not cause issues, but it's something to be aware of in dynamic lists.

Reply:
> Using the index as a key in React can lead to issues if the list changes dynamically. It's better to use a unique identifier if available. In this case, since the properties are static, it might not cause issues, but it's something to be aware of in dynamic lists.

Good point